pyintegration/deephaven2/plot/__init__.py

```diff
@@ -0,0 +1,14 @@
+#
+#   Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+#
+""" The plot package includes all the modules for creating plots. """
+
+from .axisformat import AxisFormat, DecimalAxisFormat, NanosAxisFormat
+from .axistransform import AxisTransform, axis_transform_names, axis_transform_by_name
```

Comment:
Should `axis_transform_by_name` just be `axis_transform`?  This would allow more freedom in the future, if we want to index them other ways.